rest-service/manager_rest/rest/resources_v3_1/inter_deployment_dependencies.py

```diff
@@ -0,0 +1,251 @@
+# Copyright (c) 2017-2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import uuid
+
+from flask_restful_swagger import swagger
+from flask_restful.reqparse import Argument
+
+from manager_rest.security import SecuredResource
+from manager_rest import utils, manager_exceptions
+from manager_rest.rest import rest_decorators, rest_utils
+from manager_rest.security.authorization import authorize
+from manager_rest.storage import models, get_storage_manager
+from manager_rest.utils import create_filter_params_list_description
+from manager_rest.rest.rest_utils import get_args_and_verify_arguments
+
+
+class InterDeploymentDependency(SecuredResource):
+
+    @swagger.operation(
+        responseClass=models.InterDeploymentDependencies,
+        nickname="InterDeploymentDependency",
+        notes='Return a single inter-deployment dependency',
+        parameters=create_filter_params_list_description(
+            models.InterDeploymentDependencies.response_fields,
+            'inter-deployment dependency')
+    )
+    @authorize('inter_deployment_dependency_get')
+    @rest_decorators.marshal_with(models.InterDeploymentDependencies)
+    def get(self, _include=None):
+        """Get an inter-deployment dependency by id"""
+        args = get_args_and_verify_arguments([
+            Argument('dependency_creator', required=True),
+            Argument('source_deployment', required=True),
+            Argument('target_deployment', required=True)
+        ])
+        sm = get_storage_manager()
+        dependency_creator = args['dependency_creator']
+        source_deployment = sm.get(models.Deployment,
+                                   args.get('source_deployment'))
+        target_deployment = sm.get(models.Deployment,
+                                   args.get('target_deployment'))
+        filters = {
+            'dependency_creator': args.get('dependency_creator'),
```

Comment:
The creation of this dict should be exported to util func, because it's being created in a lot of places

Reply:
Or at least const source_deployment, target_deployment and dependency_creator